Sources/SwiftFormatRules/AlwaysUseLowerCamelCase.swift

```diff
@@ -19,31 +19,72 @@ import SwiftSyntax
 /// Lint: If an identifier contains underscores or begins with a capital letter, a lint error is
 ///       raised.
 public final class AlwaysUseLowerCamelCase: SyntaxLintRule {
+  /// Stores function decls that are test cases.
+  private var testCaseFuncs = Set<FunctionDeclSyntax>()
+
+  public override func visit(_ node: SourceFileSyntax) -> SyntaxVisitorContinueKind {
+    // Tracks whether "XCTest" is imported in the source file before processing individual nodes.
+    setImportsXCTest(context: context, sourceFile: node)
+    return .visitChildren
+  }
+
+  public override func visit(_ node: ClassDeclSyntax) -> SyntaxVisitorContinueKind {
+    // Check if this class is an `XCTestCase`, otherwise it cannot contain any test cases.
```

Comment:
Since we don't have type information, this would fail to capture anyone who uses an intermediate subclass of `XCTestCase` as the base class for their tests.

Since we're checking whether `XCTest` is imported, should we just assume that that + `func test*()` is a good enough heuristic and not worry about this one?

Reply:
That seems reasonable to me. I removed the superclass check.